Project doesn't compile

> For general tech support, please see www.highcharts.com/support.
> Please report only issues about `highcharts-angular` wrapper or content of this repository.
> For general issues with Highcharts and TypeScript the correct repository to report such issues is [main Highcharts repository](https://github.com/highcharts/highcharts/issues).

### Describe the bug
A clear and concise description of what is wrong.
Got error:
Error: src/app/app.component.ts:10:13 - error TS2345: Argument of type 'typeof import(".../highcharts-angular/node_modules/highcharts/highcharts")' is not assignable to parameter of type 'typeof import(".../highcharts-angular/node_modules/highcharts/highcharts.src")'.
  Property 'apply' is missing in type 'typeof import(".../highcharts-angular/node_modules/highcharts/highcharts")' but required in type 'typeof import(".../highcharts-angular/node_modules/highcharts/highcharts.src")'.

10 SunsetTheme(Highcharts);
               ~~~~~~~~~~

  node_modules/highcharts/themes/sunset.src.d.ts:19:14
    19     function apply(): void;
                    ~~~~~
    'apply' is declared here.
src/app/app.module.ts:5:39 - error TS2307: Cannot find module 'highcharts-angular' or its corresponding type declarations.

5 import { HighchartsChartModule } from "highcharts-angular";
                                        ~~~~~~~~~~~~~~~~~~~~
I did not make any changes after cloning repository

### Expected behavior
A clear and concise description of what you expected to happen.
Project compiles and starts
### Demo
Please provide live demo of the problem or provide any other way to recreate the problem.

#### Steps required to recreate the problem in the demo:
1. Cloned repository
2. `npm install`
3. `ng serve`

### Setup used
 - NodeJS version  14.17.0
 - Angular version 11.0.5 (as specified in your repo)
 - TypeScript version 4.0.5 (as specified in your repo)

--- a/src/app/app.component.ts
+++ b/src/app/app.component.ts
@@ -2,7 +2,7 @@ declare var require: any;
 
 import { Component } from '@angular/core';
 import ExportingModule from 'highcharts/modules/exporting';
-import SunsetTheme from 'highcharts/themes/sunset.src.js';
+import SunsetTheme from 'highcharts/themes/sunset.js';
 import * as Highcharts from "highcharts";
 
 // The modules will work for all charts.

